src/components/NavBar/SearchBar.tsx

```diff
@@ -210,14 +234,14 @@ export const SearchBarDropdown = ({ toggleOpen, tokens, collections, hasInput }:
               headerIcon={<TrendingArrow />}
             />
           )}
-          {(trendingCollections?.length ?? 0) > 0 && !isTokenPage && (
+          {(trendingCollections?.length ?? 0) > 0 && !isTokenPage && phase1Flag === Phase1Variant.Enabled && (
             <SearchBarDropdownSection
               hoveredIndex={hoveredIndex}
               startingIndex={searchHistory.length + (isNFTPage ? 0 : trendingTokens?.length ?? 0)}
               setHoveredIndex={setHoveredIndex}
               toggleOpen={toggleOpen}
               suggestions={trendingCollections as unknown as GenieCollection[]}
-              header={'Trending NFT collections'}
+              header={'Popular NFT collections'}
```

Comment:
Should use `<Trans />` on string literal values

Reply:
Localization was not in the scope of this work and will be added later